<ui version="4.0" >
 <class>PrefsDialog</class>
 <widget class="QDialog" name="PrefsDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>496</width>
    <height>324</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>GGAP Preferences</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QGroupBox" name="gapBox" >
     <property name="title" >
      <string>GAP</string>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <layout class="QGridLayout" >
        <item row="1" column="0" >
         <widget class="QLabel" name="label" >
          <property name="text" >
           <string>&amp;Command:</string>
          </property>
          <property name="alignment" >
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buddy" >
           <cstring>entryGapExe</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="2" >
         <widget class="QPushButton" name="buttonGapExe" >
          <property name="text" >
           <string>...</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0" >
         <widget class="QLabel" name="label_2" >
          <property name="text" >
           <string>O&amp;ptions:</string>
          </property>
          <property name="alignment" >
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buddy" >
           <cstring>entryGapArgs</cstring>
          </property>
         </widget>
        </item>
        <item row="2" column="1" >
         <widget class="QLineEdit" name="entryGapArgs" />
        </item>
        <item row="1" column="1" >
         <widget class="QLineEdit" name="entryGapExe" />
        </item>
        <item row="0" column="0" >
         <widget class="QLabel" name="label_4" >
          <property name="text" >
           <string>&amp;Directory:</string>
          </property>
          <property name="alignment" >
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buddy" >
           <cstring>entryGapDir</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1" >
         <widget class="QLineEdit" name="entryGapDir" />
        </item>
        <item row="0" column="2" >
         <widget class="QPushButton" name="buttonGapDir" >
          <property name="text" >
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2" >
     <property name="title" >
      <string>Display</string>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <layout class="QGridLayout" >
        <item row="0" column="1" >
         <widget class="QLabel" name="label_3" >
          <property name="text" >
           <string>Worksheet font:</string>
          </property>
          <property name="alignment" >
           <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
          </property>
         </widget>
        </item>
        <item row="0" column="2" >
         <widget class="QLabel" name="labelWorksheetFont" >
          <property name="text" >
           <string>TextLabel</string>
          </property>
          <property name="alignment" >
           <set>Qt::AlignBottom|Qt::AlignHCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="3" >
         <widget class="QPushButton" name="buttonWorksheetFont" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text" >
           <string>C&amp;hoose...</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1" >
         <widget class="QLabel" name="label_5" >
          <property name="text" >
           <string>Help font:</string>
          </property>
          <property name="alignment" >
           <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
          </property>
         </widget>
        </item>
        <item row="1" column="2" >
         <widget class="QLabel" name="labelHelpFont" >
          <property name="text" >
           <string>TextLabel</string>
          </property>
          <property name="alignment" >
           <set>Qt::AlignBottom|Qt::AlignHCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="3" >
         <widget class="QPushButton" name="buttonHelpFont" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text" >
           <string>C&amp;hoose...</string>
          </property>
         </widget>
        </item>
        <item row="0" column="4" >
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="0" >
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>entryGapDir</tabstop>
  <tabstop>buttonGapDir</tabstop>
  <tabstop>entryGapExe</tabstop>
  <tabstop>buttonGapExe</tabstop>
  <tabstop>entryGapArgs</tabstop>
  <tabstop>buttonWorksheetFont</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PrefsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PrefsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
